modules/ROOT/pages/rest-api-authentication.adoc

```diff
@@ -22,7 +22,7 @@ To log in, use the following request:
 | Form Data
 | username: a username +
 password: a password +
-redirect: `true` or `false`. `false` is the default value if the redirect parameter is not specified. It indicates that the service should not redirect to Bonita Portal (after a successful login) or to the login page (after a login failure). +
+redirect: `true` or `false`. `false` is the default value if the redirect parameter is not specified. It indicates that the service should not redirect to Bonita Applications (after a successful login) or to the login page (after a login failure). +
 redirectUrl: the URL of the page to be displayed after a succesful login. If it is specified, then the a redirection after the login will be performed even if the redircet parameter is not present in the request. +
```

Comment:
if the redircet parameter => if the redirect parameter

Reply:
Added